packages/eslint-plugin-excel-custom-functions/src/rules/utils.ts

```diff
@@ -8,9 +8,9 @@ import {
 import * as ts from "typescript";
 
 export enum OfficeCalls {
-  WRITE = "WRITE", // eslint-disable-line no-unused-vars
-  READ = "READ", // eslint-disable-line no-unused-vars
-  NOTOFFICE = "NOTOFFICE", // eslint-disable-line no-unused-vars
+  WRITE = "WRITE",
```

Comment:
I think there was a bug in typescript-eslint package where no-unused-vars error message were reported, but I think it is now fixed. I am wondering if it is really desirable to have no-unused-vars set to off in the default config.

Reply:
There is the base eslint no-unused-vars and the @typescrpt-eslint/no-unused-vars.  The typescript version handles things like enums that the base rule does not.  We had rule updates in eslintrc files that disabled the base version in favor of the typescript version, but it wasn't in the linter cli.  This PR adds that to the exported configs so these disable lines are no longer needed since the typescript version covers it (verified when making the change).